packages/data-stores/src/plans/types.ts

```diff
@@ -162,6 +173,15 @@ export interface PricedAPIPlanIntroductoryOffer {
 	introductory_offer_end_date?: string;
 }
 
+export interface PricedAPIPlanCostOverrides {
```

Comment:
p.s. this being prefixed with `PricedAPIPlan` and only used in `PricedAPISitePlan` below feels a little off to me, but it's ok. The alternative would be to rename to `PricedAPISitePlanCostOverrides` (not sure, unless there is no such concept possible for the non-site plans), or map it to `PricedAPIPlanPricing` and call the `unpackCostOverrides` from both `usePlans` and `useSitePlans`.

Reply:
Please feel free to ignore